metron-stellar/stellar-common/src/test/java/org/apache/metron/stellar/dsl/functions/StringFunctionsTest.java

```diff
@@ -449,4 +450,68 @@ public void testCountMatches() throws Exception {
     Assert.assertTrue(thrown);
 
   }
+
```

Comment:
Would it be possible for you to use the 
```java
@multiline
```
Annotation for defining your json strings instead of inline concatenation and escaping?  It makes it more readable.

Reply:
Thanks @ottobackwards . Have now used multiline strings.